scripts/control_bar.js

```diff
@@ -361,6 +371,12 @@ function makeImageWidget(container, storageKey, align = "C") {
     });
 
     return {
+        setup: function (storageKey) {
+            let imageWidgetKey = storageKey + "-imagewidget";
```

Comment:
you use const above, so we could use it here too, though this is a minor comment.

Reply:
Done this.